ansible_collections/arista/avd/roles/eos_designs/templates/custom_structured_configuration/logic-router-bgp-peer-groups-struct-cfg.j2

```diff
@@ -0,0 +1,12 @@
+{% for peer_group in router_bgp.peer_groups | arista.avd.natural_sort %}
+{%     if router_bgp.peer_groups[peer_group].struct_cfg is arista.avd.defined %}
```

Comment:
This is required because other templates might add more config to the peer_group, and we want the structured_config to be the last thing applied, so it can override any config.

Reply:
Makes sense - thanks